File: src/main/java/br/com/blz/testjava/http/data/response/ProductResponse.java
package br.com.blz.testjava.http.data.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProductResponse {
    private Long sku;
    private String name;
    private InventoryResponse inventory;
    @JsonProperty(value = "isMarketable")
    private Boolean marketable;
}

File: src/main/java/br/com/blz/testjava/model/ProductResponse.kt
package br.com.blz.testjava.model

import br.com.blz.testjava.enum.TypeWarehouse
import com.fasterxml.jackson.annotation.JsonProperty

data class ProductResponse(
  val sku: Long,
  val name: String,
  val inventory: Inventory,
  @get:JsonProperty("isMarketable")
  var isMarketable: Boolean
)
